release/RELEASE.md

```diff
@@ -10,14 +10,26 @@ patches we backport to the 1.0 release branch).
 > **IMPORTANT**: If the release fails, please delete it from the [releases page]
 > and write how it failed on the PR.
 
+> **TRANSITION**: We have switched to a new release process that includes
+> Canton. In the new process, a smaller set of artifacts is generated from this
+> repo as a "split" release, then the Canton team needs to make a release based
+> on that, and then the final release is triggered from the [DACH-NY/assembly]
+> repo, referencing both this and the Canton snapshots.
+>
+> Pre-2.0.0 releases should still follow the old process, for which the steps
+> are marked **[1]**. 2.0.0 snapshots and later should follow the new process,
+> for which steps are marked **[2]**.
```

Comment:
Should have a separate document for this? I suspect that the interlaced instructions could be confusing. The old release instructions are definitely stable, so some duplication can be acceptable here, I believe.

Reply:
I agree on  the separate docs, it will be easier to follow.

Reply:
> As in `TESTING.md`, `RELEASE_1.md` and `RELEASE_2.md`?

Something along those lines, yes.